BaseListener/Boundary/EntityEventSns.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace BaseListener.Boundary
+{
+    /// <summary>
+    /// TODO
+    /// Suggested model of the event message received by the function
+    /// This will of course be dependent on the specific message expected.
+    /// </summary>
+    public class EntityEventSns
```

Comment:
candidate for nuget yet?

Reply:
Absolutely. It could probably go in the Hackney.Core.Sns package.